README.md

```diff
@@ -137,38 +162,142 @@ generate RBIs from gems
+> Are you coming from `srb rbi`? [See how `tapioca gem` compares to `srb rbi`](https://github.com/Shopify/tapioca/wiki/How-does-tapioca-compare-to-%22srb-rbi-gems%22-%3F).
 
-Command: `tapioca todo`
+#### Manually requiring parts of a gem
 
-This will generate the file `sorbet/rbi/todo.rbi` defining all unresolved constants as empty modules.
+It may happend that the RBI file generated for a gem inside your `Gemfile.lock` is missing some definitions the gem seems to be exporting.
+
+For gems that have a normal default `require` and load all of their constants through such a require, everything should work seamlessly. However, for gems that are marked as `require: false` in the Gemfile, or for gems that export constants optionally via different requires, where a single require does not load the whole gem code into memory, Tapioca will not be able to load some of the types into memory and, thus, won't be able to generate complete RBIs for them. For this reason, we need to keep a small external file named `sorbet/tapioca/require.rb` that is executed after all the gems in the Gemfile have been required and before generation of gem RBIs have started. This file is responsible for adding the requires for additional files from gems, which are not covered by the default require.
+
+For example, suppose you are using the class `BetterHtml::Parser` exported from the `better_html` gem. Just doing a `require "better_html"` (which is the default require) does not load that type:
 
-<!-- START_HELP_COMMAND_TODO -->
 ```shell
-Usage:
-  tapioca todo
+$ bundle exec pry
 
-Options:
-      [--todo-file=TODO_FILE]              # Path to the generated todo RBI file
-                                           # Default: sorbet/rbi/todo.rbi
-      [--file-header], [--no-file-header]  # Add a "This file is generated" header on top of each generated RBI file
-                                           # Default: true
-  -c, [--config=<config file path>]        # Path to the Tapioca configuration file
-                                           # Default: sorbet/tapioca/config.yml
-  -V, [--verbose], [--no-verbose]          # Verbose output for debugging purposes
+[1] pry(main)> require 'better_html'
+=> true
+[2] pry(main)> BetterHtml
+=> BetterHtml
+[3] pry(main)> BetterHtml::Parser
+NameError: uninitialized constant BetterHtml::Parser
+from (pry):3:in `__pry__`
+[4] pry(main)> require 'better_html/parser'
+=> true
+[5] pry(main)> BetterHtml::Parser
+=> BetterHtml::Parser
+```
 
-generate the list of unresolved constants
+In order to make sure that `tapioca` can reflect on that type, we need to add the line `require "better_html/parser"` to the `sorbet/tapioca/require.rb` file. This will make sure `BetterHtml::Parser` is loaded into memory and a type annotation is generated for it in the `better_html.rbi` file. If this extra `require` line is not added to `sorbet/tapioca/require.rb` file, then the definition for that type will be missing from the RBI file.
```

Comment:
```suggestion
In order to make sure that `tapioca` can reflect on that type, we need to add the line `require "better_html/parser"` to the `sorbet/tapioca/require.rb` file. This will make sure `BetterHtml::Parser` is loaded into memory and a type annotation is generated for it in the `better_html.rbi` file. If this extra `require` line is not added to `sorbet/tapioca/require.rb` file, then Tapioca will be able to generate definitions for `BetterHtml` and other constants, but not for `BetterHtml::Parser`, which will be missing from the RBI file.
```